c/pthread/bounded_buffer_false-unreach-call.c

```diff
@@ -0,0 +1,377 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+extern int __VERIFIER_nondet_int(void);
+
+#include <stdio.h>
```

Comment:
How can I produce the preprocess files? Could you please send me the command line?